src/containers/Sidebar.js

```diff
@@ -0,0 +1,10 @@
+import { connect } from 'react-redux';
+import { getUserLocation, registerUserLocation } from '../actions/index';
+import Sidebar from '../components/Sidebar';
+
+const mapDispatchToProps = dispatch => ({
+  // registerUserLocation: () => dispatch(registerUserLocation()),
```

Comment:
This comment is necessary or not?

Reply:
I commented this out, because once we merge backend and this user frontend, i need to store the geocode to the db. i just prepared for it